From d7d8cfbb34bec344cd8cbe5bdd864c588f62a463 Mon Sep 17 00:00:00 2001
From: Mete Alpaslan <mete@pardus.org.tr>
Date: Thu, 2 Dec 2010 16:05:33 +0200
Subject: [PATCH 5/5] yali.postinstall:Use comar in setting hostname also.

Use hostname command to set hostname instantly and use comar to write
in /etc/env.d and after restart.
---
 yali/postinstall.py |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/yali/postinstall.py b/yali/postinstall.py
index a89553f..5bb520d 100644
--- a/yali/postinstall.py
+++ b/yali/postinstall.py
@@ -102,6 +102,7 @@ def setHostName():
     obj = bus.get_object("tr.org.pardus.comar", "/package/baselayout")
     if ctx.flags.install_type == ctx.STEP_FIRST_BOOT:
         yali.util.run_batch("hostname", [str(ctx.installData.hostName)])
+        obj.setHostName(str(ctx.installData.hostName), dbus_interface="tr.org.pardus.comar.Network.Stack")
     elif ctx.flags.install_type == ctx.STEP_DEFAULT:
         obj.setHostName(str(ctx.installData.hostName), dbus_interface="tr.org.pardus.comar.Network.Stack")
     elif ctx.flags.install_type == ctx.STEP_BASE:
-- 
1.7.2.2

